src/components/TodoList/TodoList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
+type Todos = {
+  todoList: Todo[],
+  setSelectedTodoId: (userId: number) => void,
+  selectedTodoId: number,
+};
+
+export const TodoList: React.FC<Todos> = (
+  {
+    todoList, setSelectedTodoId, selectedTodoId,
+  },
```

Comment:
It'll look much better
```suggestion
export const TodoList: React.FC<Todos> = (
  {
    todoList, 
    setSelectedTodoId, 
    selectedTodoId,
  },
```